pkg/synopsysctl/cmd_destroy.go

```diff
@@ -88,9 +92,26 @@ func destroy(namespace string) {
 	}
 
 	// delete namespace
-	err = util.DeleteResourceNamespace(restconfig, kubeClient, strings.Join(crds, ","), namespace, true)
-	if err != nil {
-		log.Warn(err)
+	checkErr := util.CheckResourceNamespace(restconfig, kubeClient, strings.Join(crds, ","), namespace, true)
+	if err != nil && isForceDestroy {
+		log.Warnf("%s. namespace cannot be deleted", err.Error())
+	} else if err != nil && !isForceDestroy {
+		return fmt.Errorf("%s. we do not recommend destroying the Synopsys Operator for proper upkeep of these resources. you can use 'force' option, which will keep the instances, but delete the Synopsys Operator.  You can re-deploy the Synopsys Operator in this namespace at any time", err.Error())
```

Comment:
we do not ...-> it is not recommended to destroy the Synopsys Operator so these resources can continue to be managed.  If you are sure you want to delete the Synopsys Operator anyway then you can use the 'force' option which will keep all the instances and delete only the Synopsys Operator. ...

Reply:
@rrati done Rob